common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -437,6 +437,12 @@ workflow_state = ? ` +
 		`and visibility_ts = ? ` +
 		`and task_id = ?`
 
+	templateListCurrentExecutionsQuery = `SELECT domain_id, workflow_id, current_run_id ` +
+		`FROM executions ` +
+		`WHERE shard_id = ? ` +
+		`and type = ? ` +
+		`and run_id = ?`
```

Comment:
Tried this query on my local machine and got 
```
InvalidRequest: Error from server: code=2200 [Invalid query] message="PRIMARY KEY column "run_id" cannot be restricted as preceding column "domain_id" is not restricted"
```
Add a TODO here?

Reply:
good catch. we actually need to filter on domainID - we have a cookie cutter domainID for currentExecutions

Reply:
Please correct me if I am wrong. The primary key for the table is `(shard_id, type, domain_id, workflow_id, run_id, visibility_ts, task_id)`. If we have filter on `run_id`, then I think all previous columns need to be filtered, which means we also need to add a filter on `workflow_id`.